### PR TITLE
build: added new component to list the ordered, unordered list

### PR DESCRIPTION
**Pull request by BaskarMitrah**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2265,6 +2272,47 @@ AccordionItem default expand
...
+
+Use `slots` to identify the markdown content:
+
+- `left`(required)
+
+- `right`(required)
+
+Use `repeat` to define how many code sections are part.
+
+Use `list` to define the order or unorder list. Defaults to `unorder`.
```
Comment (louisachu) [inline README.md:2309]: What does order and unorder actually do?  It seems like it only change the icons of the list if it's order.  It'll default the icon to number if there is no icon specified.  Please see icon for comments.

File: README.md [hunk 2]
```diff
@@ -2265,6 +2272,47 @@ AccordionItem default expand
...
+
+Use `iconColor` to define the marker color. The preset is black.
+
+Use `icon` used to indicate individual items or elements within the list. If the list is in order by default it is `disc` else `number`. Values are `checkmark,disc, number`
```
Comment (louisachu) [inline README.md:2313]: It seems like the default of the order list is number and not disc.  Also, order list seems to be not able to take in checkmark.  If this is by design, we may want to make sure the user knows about this.  For unorder list, the default is disc.  We may want to make this clear to the user as well.

File: README.md [hunk 3]
```diff
@@ -2265,6 +2272,47 @@ AccordionItem default expand
...
+Scalable for high volume needs.
+```
+
+Use `slots` to identify the markdown content:
+
+- `left`(required)
+
+- `right`(required)
+
+Use `repeat` to define how many code sections are part.
```
Comment (louisachu) [inline README.md:2307]: 'repeat' seems to be a required.  Please make sure this is added in readme.

File: README.md [hunk 4]
```diff
@@ -2265,6 +2272,47 @@ AccordionItem default expand
...
+Easy to sign up and create credentials in minutes
+
+Technical Support included (different tiers available)
+
+No credit card or commitment required
+
+Scalable for high volume needs.
+```
+
+Use `slots` to identify the markdown content:
```
[the remark this answers is not in the document]
Comment (BaskarMitrah) [reply README.md:2301]: I added some content information on the README

File: README.md (hunk 2 above)
Comment (BaskarMitrah) [reply README.md]: I removed the ordered/unordered list

File: README.md (hunk 3 above)
Comment (BaskarMitrah) [reply README.md]: I added as required

**Review by louisachu — APPROVED** [review]
Looks good to me

**Pull request merged**